3rdparty/libgcov/CMakeLists.txt

```diff
@@ -1,7 +1,19 @@
 # Copyright (c) Open Enclave SDK contributors.
 # Licensed under the MIT License.
 
-check_submodule_not_empty(compiler-rt)
+# Copy the following compiler-rt files to the current directory.
```

Comment:
We can consider directly building the libgcov as part of compiler-rt folder, avoiding these copy operations.